test.sh

```diff
@@ -236,6 +238,17 @@ clean_up()
     fi
 }
 
+convert_notebooks()
+{
+    echo "testing notebooks"
+    for notebook in `grep ipynb docs/tutorials.rst | sed -e 's/^[ \t]*//'`
+    do
+    jupyter nbconvert --to notebook --execute "docs/$notebook"
+    check_errs $?
```

Comment:
```suggestion
```
I think the line shown above should be removed here as the function `check_errs` is for pytest.

Reply:
Actually, I would keep it and change:

https://github.com/TDAmeritrade/stumpy/blob/575a9a49efa09dfe049a2c1fc55f35642d5d308b/test.sh#L39

to

```
echo "Error: Test execution encountered exit code $1"
```

The `check_errs` function isn't `pytest` specific and simply checks whether or not the last executed command terminated "successfully". If not, the script itself should also "die" without proceeding any further

Reply:
Yeah...I actually meant the message (inside that function) was about pytest. I should have spoken more clearly. I can see your point. I will revise the line.